dc-model/src/main/java/de/digitalcollections/model/list/buckets/BucketsResponse.java

```diff
@@ -0,0 +1,33 @@
+package de.digitalcollections.model.list.buckets;
+
+import de.digitalcollections.model.UniqueObject;
+import de.digitalcollections.model.list.ListResponse;
+import java.util.List;
+
+public class BucketsResponse<T extends UniqueObject>
+    extends ListResponse<Bucket<T>, BucketsRequest<T>> {
+
+  private BucketsRequest<T> bucketsRequest;
```

Comment:
```suggestion

```
Das sollte das Feld `request` beinhalten (in der Klasse `ListResponse`).

Reply:
done